docs/application/dotnet/guides/nui/text.md

```diff
@@ -402,6 +402,25 @@ field.HorizontalAlignment = HorizontalAlignment.Begin;
 ```
 
 <a name="textField4"></a>
+### Use Input properties in TextField
+
+Input properties, such as `InputColor` and `InputPointSize`, are used when you want to set new input texts differently from the existing text settings when the new text inputs.
+
+Especially, these properties should be used with some specific conditions.
+
+For instance, they're used when `TextChanged` signal is emitted or a button is clicked.
+
+The following example illustrates how to use Input properties:

```

Comment:
The following example illustrates how to use the Input properties:

Reply:
Done.